sentry/templates/hooks/clickhouse-init.job.yaml

```diff
@@ -38,7 +39,9 @@ spec:
           - -ec
           - >-
             for ((i=0;i<{{ .Values.clickhouse.clickhouse.replicas }};i++)); do
+              clickhouse-client --database=default --host={{ $clickhouseHost }}-$i.{{ $clickhouseHost }}-headless --port={{ $clickhousePort }} --query='{{ $dropQuery }}';
```

Comment:
Does it mean we drop the table every time we run the hooks?

Reply:
Yes, however the dustributed table only contains the structure and a reference to the local table where the data is actually stored so no data loss occurs.